src/utils.rs

```diff
@@ -39,6 +38,24 @@ pub async fn configure_from_env(ctx: &Context) -> Result<()> {
     Ok(())
 }
 
+/// Returns the path to assets directory.
+///
+/// Searches for "assets" directory near the bot binary.
+/// If it does not exist, assumes that it is a development setup
+/// and "assets" directory can be found in the current working directory.
+pub(crate) fn assets_path() -> Result<PathBuf> {
+    let exe = std::env::current_exe().context("cannot determine executable path")?;
+    let dir = exe
+        .parent()
+        .with_context(|| format!("cannot determine binary directory for {}", exe.display()))?
+        .join("assets");
+    if dir.exists() {
+        Ok(dir)
+    } else {
+        Ok(PathBuf::from("assets"))
+    }
+}
+
```

Comment:
I personally like to use `#[cfg(debug_assertions)]` to make such decisions. Breaks running `cargo r --release` but that should be fine imo.